# This file is @generated, regenerate by re-running test with `-- --env BUCK2_UPDATE_GOLDEN=1` appended to the test command

{
  "failures": {
    "root//fail_action:fail_one": "Action failed: root//fail_action:fail_one (root//config:dog_platform#<HASH>) (failname)\nRemote command returned non-zero exit code 1\nRemote action, reproduce with: `frecli cas download-action <DIGEST>`\nStdout: <empty>\nStderr: <empty>\n",
    "root//fail_analysis:fail_analysis": "Error running analysis for `root//fail_analysis:fail_analysis (<unspecified>)`\n\nCaused by:\n    Traceback (most recent call last):\n      File <builtin>, in <module>\n      * fail_analysis/fail_analysis.bzl:9, in _impl\n          fail(\"Failure during analysis\")\n    error: fail: Failure during analysis\n     --> fail_analysis/fail_analysis.bzl:9:5\n      |\n    9 |     fail(\"Failure during analysis\")\n      |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n      |\n    ",
    "root//fail_config:cat_only": "Error running analysis for `root//config:not_a_target_platform (<unbound>)`\n\nCaused by:\n    0: Error looking up configured node root//config:not_a_target_platform (<unbound>)\n    1: looking up unconfigured target node `root//config:not_a_target_platform`\n    2: Unknown target `not_a_target_platform` from package `root//config`.\n       Did you mean one of the 5 targets in root//config:TARGETS.fixture?",
    "root//fail_load:first": "Error evaluating build file: `root//fail_load:TARGETS.fixture`\n\nCaused by:\n    Traceback (most recent call last):\n      * fail_load/TARGETS.fixture:1, in <module>\n          fail(\"Non-specific load failure\")\n    error: fail: Non-specific load failure\n     --> fail_load/TARGETS.fixture:1:1\n      |\n    1 | fail(\"Non-specific load failure\")\n      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n      |\n    ",
    "root//missing:missing": "Unknown target `missing` from package `root//missing`.\nDid you mean one of the 1 targets in root//missing:TARGETS.fixture?"
  },
  "results": {
    "root//fail_action:fail_one": {
      "configured": {
        "root//config:dog_platform#<HASH>": {
          "configured_graph_size": null,
          "errors": [
            {
              "action_error": {
                "digest": "<DIGEST>",
                "error_content": "<STRING_HASH>",
                "error_diagnostics": null,
                "key": {
                  "owner": "root//fail_action:fail_one (root//config:dog_platform#<HASH>)"
                },
                "name": {
                  "category": "failname",
                  "identifier": ""
                },
                "stderr_content": "<STRING_HASH>",
                "stdout_content": "<STRING_HASH>"
              },
              "cause_index": 0,
              "error_tags": [
                "ACTION_COMMAND_FAILURE",
                "ANY_ACTION_EXECUTION"
              ],
              "message_content": "<STRING_HASH>"
            }
          ],
          "other_outputs": {},
          "outputs": {},
          "success": "FAIL"
        }
      },
      "configured_graph_size": null,
      "errors": [],
      "other_outputs": {},
      "outputs": {},
      "success": "FAIL"
    },
    "root//fail_analysis:fail_analysis": {
      "configured": {
        "<unspecified>": {
          "configured_graph_size": null,
          "errors": [
            {
              "action_error": null,
              "cause_index": 1,
              "error_tags": [
                "ANALYSIS",
                "STARLARK_FAIL"
              ],
              "message_content": "<STRING_HASH>"
            }
          ],
          "other_outputs": {},
          "outputs": {},
          "success": "FAIL"
        }
      },
      "configured_graph_size": null,
      "errors": [],
      "other_outputs": {},
      "outputs": {},
      "success": "FAIL"
    },
    "root//fail_config:cat_only": {
      "configured": {},
      "configured_graph_size": null,
      "errors": [
        {
          "action_error": null,
          "cause_index": 2,
          "error_tags": [
            "ANALYSIS",
            "MISSING_TARGET"
          ],
          "message_content": "<STRING_HASH>"
        }
      ],
      "other_outputs": {},
      "outputs": {},
      "success": "FAIL"
    },
    "root//fail_load:first": {
      "configured": {},
      "configured_graph_size": null,
      "errors": [
        {
          "action_error": null,
          "cause_index": 3,
          "error_tags": [
            "STARLARK_FAIL"
          ],
          "message_content": "<STRING_HASH>"
        }
      ],
      "other_outputs": {},
      "outputs": {},
      "success": "FAIL"
    },
    "root//missing:missing": {
      "configured": {},
      "configured_graph_size": null,
      "errors": [
        {
          "action_error": null,
          "cause_index": 4,
          "error_tags": [
            "MISSING_TARGET"
          ],
          "message_content": "<STRING_HASH>"
        }
      ],
      "other_outputs": {},
      "outputs": {},
      "success": "FAIL"
    },
    "root//success:success": {
      "configured": {
        "<unspecified>": {
          "configured_graph_size": null,
          "errors": [],
          "other_outputs": {},
          "outputs": {},
          "success": "SUCCESS"
        }
      },
      "configured_graph_size": null,
      "errors": [],
      "other_outputs": {},
      "outputs": {},
      "success": "SUCCESS"
    }
  },
  "strings": {
    "<STRING_HASH>": "",
    "<STRING_HASH>": "Action failed: root//fail_action:fail_one (root//config:dog_platform#<HASH>) (failname)\nRemote command returned non-zero exit code 1\nRemote action, reproduce with: `frecli cas download-action <DIGEST>`\nStdout: <empty>\nStderr: <empty>\n",
    "<STRING_HASH>": "Error evaluating build file: `root//fail_load:TARGETS.fixture`\n\nCaused by:\n    Traceback (most recent call last):\n      * fail_load/TARGETS.fixture:1, in <module>\n          fail(\"Non-specific load failure\")\n    error: fail: Non-specific load failure\n     --> fail_load/TARGETS.fixture:1:1\n      |\n    1 | fail(\"Non-specific load failure\")\n      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n      |\n    ",
    "<STRING_HASH>": "Error running analysis for `root//config:not_a_target_platform (<unbound>)`\n\nCaused by:\n    0: Error looking up configured node root//config:not_a_target_platform (<unbound>)\n    1: looking up unconfigured target node `root//config:not_a_target_platform`\n    2: Unknown target `not_a_target_platform` from package `root//config`.\n       Did you mean one of the 5 targets in root//config:TARGETS.fixture?",
    "<STRING_HASH>": "Error running analysis for `root//fail_analysis:fail_analysis (<unspecified>)`\n\nCaused by:\n    Traceback (most recent call last):\n      File <builtin>, in <module>\n      * fail_analysis/fail_analysis.bzl:9, in _impl\n          fail(\"Failure during analysis\")\n    error: fail: Failure during analysis\n     --> fail_analysis/fail_analysis.bzl:9:5\n      |\n    9 |     fail(\"Failure during analysis\")\n      |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n      |\n    ",
    "<STRING_HASH>": "Remote command returned non-zero exit code 1",
    "<STRING_HASH>": "Unknown target `missing` from package `root//missing`.\nDid you mean one of the 1 targets in root//missing:TARGETS.fixture?"
  },
  "success": false,
  "truncated": false
}
